manuscript.md

```diff
@@ -37,104 +37,81 @@ We will also explore how other omics fields are dealing with OSS and FAIR softwa
-| 6. **Develop thorough documentation.** |
-|   - User documentation: Provide tutorials, installation guides, and usage examples that lower barriers to entry for new users. |
-|   - Developer documentation: Include technical details that make it easier for new developers to understand the codebase, contribute, and debug. |
-|   - Version control and changelog: Maintain a detailed changelog for tracking updates, and consider using semantic versioning for releases to help users track changes and updates. |
-| 7. **Build a community.** Create forums, mailing lists, or a Slack channel to facilitate communication and support for users and contributors. Promote the project within academic and industry circles, social media, or conferences. Collaborations with other researchers can boost credibility and attract users. Encourage diverse participation, whether from seasoned developers, scientists, or students, by being open to questions, feedback, and contributions of varying levels. |
-| 8. **Ensure long-term maintenance and evolution.** Provide a roadmap that outlines planned features, updates, or long-term project goals. This helps maintainers and contributors stay aligned and gives users confidence in the project’s development. Foster a healthy, engaged community by recognizing contributors, hosting hackathons or sprints, and encouraging new ideas. Consider a governance model where a core group of maintainers or a steering committee manages long-term development, ensuring that the project's mission endures even as individual contributors come and go. |
-| 9. **Monitor and measure success.** Track metrics like repository stars, downloads, citations, or code contributions to gauge adoption and impact. Regularly collect user feedback and address concerns or feature requests to ensure the project stays relevant and useful to its audience. |
-[W: Open to discussion. Please carefully double-check whether these steps make sense and that I haven't forgotten anything.] [B: 7-9 is often not possible in our current funding system - we should make clear that the first bullet points are a must have and the last 3 are a goal][M: A major problem is that there are very very few examples of this in proteomics ... there are a handful tops. I think this needs to be toned down a lot. This should be a goal but the funding agencies have to be supportive of this first. The issue is that the NIH doesn't want to fund open source software development and they definitely don't want to fund repositories.
-]
+- **Freedom to Modify and Distribute**: Open-source software allows users not only to access the code but also to modify it and share these modifications, encouraging innovation and collaboration.
```

Comment:
Could clarify some confusion about open source software vs source-available but no modifications can be me (non-OSI):

"Open-source software, in contrast to source-available software, allows users ..."